full-moon/src/ast/versions.rs

```diff
@@ -121,6 +122,25 @@ impl LuaVersion {
     pub fn has_luajit(self) -> bool {
         cfg!(feature = "luajit") && (self.bitfield & VERSION_LUAJIT != 0)
     }
+
+    /// Creates a new LuaVersion with only CFX Lua.
+    #[cfg(feature = "cfxlua")]
+    pub fn cfxlua() -> Self {
+        Self {
+            bitfield: VERSION_CFXLUA,
```

Comment:
Similar to the `lua54` example above, since cfxlua is an extension of lua54, we should extend the bitfield
```suggestion
            bitfield: VERSION_LUA52 | VERSION_LUA53 | VERSION_LUA54 | VERSION_CFXLUA,
```